grep-9999-grc-out-of-tree.md

```diff
@@ -0,0 +1,164 @@
+# GREP [XXXX] -- [Move GRC Out of Gree]
```

Comment:
GREE -> GR Tree?

Reply:
it's a convenient abbreviation :man_facepalming: